### PR TITLE
refactor: Test and Clean Up Store

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
 * GDT balance is no longer stored in store. It is a property of the DashboardLayout (as balance)
 * Testing the store
 * As create user and login do the same, just login is used.
 * Coverage to 15%

### CODE REVIEW

File: frontend/src/store/store.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        expect(commit).toHaveBeenNthCalledWith(1, 'session_id', null)
+      })
+
+      it('commits email', () => {
+        logout({ commit, state })
+        expect(commit).toHaveBeenNthCalledWith(2, 'email', null)
+      })
+
+      // how can I get this working?
+      it.skip('calls sessionStorage.clear()', () => {
```
Comment (Mogge) [inline frontend/src/store/store.test.js:74]: I spent nearly two hours, But I cannot get this working :(

File: frontend/src/views/Pages/Register.vue [hunk 2]
```diff
@@ -174,7 +174,7 @@ export default {
         this.password,
       )
       if (result.success) {
-        this.$store.dispatch('createUser', {
+        this.$store.dispatch('login', {
```
Comment (Mogge) [inline frontend/src/views/Pages/Register.vue:177]: I think we do not need this. After register, the user is redirected to the thank page. There is a link to login. So way store a session id when the user has to login again and the receives as session id? Does not make sense to me.

File: frontend/src/views/Layout/DashboardLayout_gdd.vue [hunk 3]
```diff
@@ -92,6 +97,14 @@ export default {
         // what to do when loading balance fails?
       }
     },
+    async loadGDTBalance() {
+      const result = await communityAPI.transactions(this.$store.state.session_id)
+      if (result.success) {
+        this.GdtBalance = result.result.data.gdtSum
```
Comment (Mogge) [inline frontend/src/views/Layout/DashboardLayout_gdd.vue:103]: / 10000 ??

**Review by ulfgebhardt — APPROVED** [review]
Looks good <3

**Pull request merged**